metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -34,6 +34,30 @@
 GMS_SERVER = f"http://localhost:{GMS_PORT}"
 
```

Comment:
can you also add some unit tests for the `merge_refinement` method? These should be totally separate from the integration tests here

Reply:
It was time consuming as existing code has dependency on Pipeline context etc... 

let me try to add

Reply:
the `merge_refinement` is a static method and hence does not have any dependency on pipeline context or config

Reply:
Done. Added a test-case